kits/daisy/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+cmake_minimum_required (VERSION 3.1)
+project (hexapod)
+
+SET (CMAKE_CXX_STANDARD 11)
+SET (CMAKE_CXX_STANDARD_REQUIRED ON)
+SET (CMAKE_CXX_FLAGS "-O3 -g")
+SET (SONAME "hebi")
+
+# Instruct CMake to run moc automatically when needed.
+set(CMAKE_AUTOMOC ON)
+
+# Find Qt libraries
+find_package(Qt5 REQUIRED COMPONENTS Core Gui Widgets NO_CMAKE_PATH)
+# NOTE: to add support for 3D Qt components, replace the above with the
+# following line
+#find_package(Qt5 REQUIRED COMPONENTS Core Gui Widgets 3DCore 3DExtras 3DRender 3DInput NO_CMAKE_PATH)
+
+find_package(Doxygen)
+
+# Get the C++ API directory:
+set(ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+get_filename_component(ROOT_DIR "${ROOT_DIR}" REALPATH)
+# `HEBI_CPP_DIR` can be used to point to an already downloaded C++ API.
+# This is useful for environments that would prefer not to download files
+# from the internet (or even don't have internet access).
+if(NOT HEBI_CPP_DIR)
+  set(HEBI_DIR ${ROOT_DIR}/hebi-cpp)
+  include(../../projects/cmake/DownloadHebiCpp.cmake)
```

Comment:
Why not use `ROOT_DIR` which was created above instead of assuming the working directory is `${CMAKE_CURRENT_SOURCE_DIRECTORY}`?

Reply:
In regards to all the CMake comments -- completely agree this needs a pass.  I took the existing CMake file, and updated it to use the common `hebi-cpp` folder, but I didn't do any other cleanup yet.

FWIW, this code hasn't been made/tested as Windows compliant at all yet; I'll add an issue for that + the cmake refactoring.

Reply:
re: Windows, that's completely fine. We can fix that in the upcoming weeks.